.github/workflows/CI.yml

```diff
@@ -0,0 +1,109 @@
+# Run CI tests
+name: CI
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master, develop ]
```

Comment:
This repo has a `main`, but not `master`. Does `main` need to be included in these lists?

Reply:
Yeah that's a typo. Thanks. Fixed.